include/seqan3/alignment/aligned_sequence/debug_stream_alignment.hpp

```diff
@@ -61,19 +63,19 @@ void stream_alignment(debug_stream_type<char_t> & stream, alignment_t const & al
         // write first sequence
         stream << '\n' << std::setw(8) << "";
         std::ranges::for_each(get<0>(align) | views::slice(begin_pos, end_pos) | views::to_char,
-                              [&stream] (char ch) { stream << ch; });
+                              [&stream](char ch) { stream << ch; });
```

Comment:
can we keep the space?

Reply:
clang-format has no options for this. it will always be `[]() {}`